dockerizeit/slave/Dockerfile

```diff
@@ -22,6 +22,9 @@ RUN apk update && apk add py-pip && pip install docker-compose==1.7 docopt==0.6.
 RUN curl --create-dirs -sSLo /usr/share/jenkins/swarm-client-$JENKINS_SWARM_VERSION-jar-with-dependencies.jar http://maven.jenkins-ci.org/content/repositories/releases/org/jenkins-ci/plugins/swarm-client/$JENKINS_SWARM_VERSION/swarm-client-$JENKINS_SWARM_VERSION-jar-with-dependencies.jar \
   && chmod 755 /usr/share/jenkins
 
+#Install tool to do clean up of all unused docker layers and images
+RUN curl -s https://raw.githubusercontent.com/ZZROTDesign/docker-clean/v2.0.4/docker-clean | tee /usr/local/bin/docker-clean > /dev/null && chmod +x /usr/local/bin/docker-clean
```

Comment:
@alexsedova let's fork it to make sure that we have a copy and then use that copy